statsmodels/tsa/statespace/mlemodel.py

```diff
@@ -1113,6 +1113,199 @@ def zvalues(self):
         """
         return self.params / self.bse
 
+    def test_normality(self):
+        """
+        Jarque-Bera test for normaility of standardized residuals.
+
+        Null hypothesis is normality.
+
+        Notes
+        -----
+        If the first `d` loglikelihood values were burned (i.e. in the
+        specified model, `loglikelihood_burn=d`), then this test is calculated
+        ignoring the first `d` residuals.
+
+        See Also
+        --------
+        statsmodels.stats.stattools.jarque_bera
+
+        """
+        from statsmodels.stats.stattools import jarque_bera
+        d = self.loglikelihood_burn
+        return np.array(jarque_bera(
+            self.filter_results.standardized_forecasts_error[:, d:],
+            axis=1
+        )).transpose()
```

Comment:
Question
If endog is multivariate, then the forecast_error didn't remove correlation, only divided by standard deviation, did it?

We might need a `method` kewyword in some of these tests. 
For forwards compatibility I might even like to force users to specify a method keyword.
I'm thinking mainly of the case where we will get better normality tests for the multivariate case.